src/toil/wdl/toilwdl.py

```diff
@@ -141,7 +142,8 @@ def main():
         sWDL.write_mappings(aWDL)
     else:
         wdllogger.debug('WDL file compiled to toil script.  Running now.')
-        cmd = ['python', sWDL.output_file]
+        exe = sys.executable
```

Comment:
In some cases, `sys.executable` will be `None`, so this should be:

`exe = sys.executable if sys.executable else 'python'`

Otherwise, lgtm.

Reply:
I will update this PR. Glad you spotted that.